### ISSUE TITLE
Module could not be found

### ISSUE THREAD

**Issue by HipPenguin123** [R2Northstar/Northstar#170]
If I try and run the launcher file it says that the module could not be found and closes the tab. Not sure what's going on exactly 

  - Northstar version: 1.4.0
  - Platform: Origin

**Comment by x3Karma** [comment]
Could you not open a new issue and check #faq in discord?

It seems likely that you installed Northstar in an incorrect directory.

**Comment by GeckoEidechse** [comment]
Additionally, please provide screenshots to further help us investigate the issue.

**Comment by GeckoEidechse** [comment]
[Does this section in the wiki](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting#failed-to-load-the-tier0.dll-at-less-than-file-location-greater-than.-the-specified-module-could-not) describe your issue?

**Comment by MiloAkerman** [comment]
@GeckoEidechse could a label for "possibly resolved" be added for issues waiting on author input

**Comment by abarichello** [comment]
That's what 'info-needed' is for, I'm closing this since the issue template wasn't followed
